packages/marketplace/src/components/pages/client.tsx

```diff
@@ -74,19 +63,32 @@ export const handleInstallationDone = ({
   }
 }
 
-export type ClientProps = ClientMappedActions & ClientMappedProps & RouteComponentProps<{ page?: any }>
+export const handleSetStateViewBrowse = (dispatch: Dispatch) => () => {
+  dispatch(setAppDetailModalStateBrowse())
+}
+
+export const handleFetchAppDetail = (dispatch: Dispatch) => (id, clientId) => {
+  dispatch(appDetailRequestData({ id, clientId }))
+}
+
+export const handleInstallationsSetFormState = (dispatch: Dispatch) => (formState: FormState) => {
+  dispatch(appInstallationsSetFormState(formState))
+}
+
+export const Client: React.FunctionComponent = () => {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const location = useLocation()
+
+  const appSummaryState = useSelector(selectAppSummary)
+  const appDetail = useSelector(selectAppDetail)
+  const clientId = useSelector(selectClientId)
+  const installationsFormState = useSelector(selectInstallationFormState)
+
+  const setStateViewBrowse = handleSetStateViewBrowse(dispatch)
+  const fetchAppDetail = handleFetchAppDetail(dispatch)
+  const installationsSetFormState = handleInstallationsSetFormState(dispatch)
 
-export const Client: React.FunctionComponent<ClientProps> = ({
-  appSummaryState,
-  history,
-  location,
-  fetchAppDetail,
-  appDetail,
-  clientId,
-  setStateViewBrowse,
-  installationsFormState,
-  installationsSetFormState,
-}) => {
   const pageNumber =
```

Comment:
For page you can use like this
![image](https://user-images.githubusercontent.com/22660833/82865528-24fdc680-9f51-11ea-963a-d6151ce7d9d7.png)

Reply:
Pls remove `Number`, because page also have type number and everyone also know that